.web-docs/components/builder/docker/README.md

```diff
@@ -281,6 +303,96 @@ You must specify (only) one of `commit`, `discard`, or `export_path`.
   login, login_username, and login_password will be ignored. For more
   information see the section on ECR.
 
+<!-- End of code generated from the comments of the Config struct in builder/docker/config.go; -->
+
+
+<!-- Code generated from the comments of the AwsAccessConfig struct in builder/docker/ecr_login.go; DO NOT EDIT MANUALLY -->
```

Comment:
Can you move this change into a seperate PR? It adds a lot of lines to the diff and makes this PR a bit bigger, I think this AWS docs change is good to approve right away also

Reply:
Yeah that can be split indeed; I had changed it to account for the new partials since the Docker docs were verbatim in the README.mdx instead of including them, so that can be done in a separate PR.
I'll open it ASAP